homestead.root/usr/share/clearwater/bin/poll_homestead-prov.sh

```diff
@@ -34,22 +34,7 @@
 # under which the OpenSSL Project distributes the OpenSSL toolkit software,
 # as those licenses appear in the file LICENSE-OPENSSL.
 
-# This script uses HTTP to poll a homestead-prov process and check whether it is healthy.
-
-# In case homestead-prov has only just restarted, give it a few seconds to come up.
-sleep 5
-
-# Send HTTP request and check that the response is "OK".
-http_url=http://127.0.0.1:8889/ping
-curl -f -g -m 2 -s $http_url 2> /tmp/poll-homestead_prov.sh.stderr.$$ | tee /tmp/poll-homestead_prov.sh.stdout.$$ | head -1 | egrep -q "^OK$"
-rc=$?
-
-# Check the return code and log if appropriate.
-if [ $rc != 0 ] ; then
-  echo HTTP failed to $http_url             >&2
-  cat /tmp/poll-homestead_prov.sh.stderr.$$ >&2
-  cat /tmp/poll-homestead_prov.sh.stdout.$$ >&2
-fi
-rm -f /tmp/poll-homestead_prov.sh.stderr.$$ /tmp/poll-homestead_prov.sh.stdout.$$
-
-exit $rc
+. /etc/clearwater/config
+http_ip=$(/usr/share/clearwater/bin/bracket_ipv6_address.py $local_ip)
+/usr/share/clearwater/bin/poll-http $http_ip:8889
```

Comment:
In which namespace does poll-http run?  For homestead, it looks like it's running in the signaling namespace, but here it's running in the management namespace (I hope).  Have you tested with multiple namespaces?